packages/playground/src/components/funds_card.vue

```diff
@@ -25,31 +25,29 @@
 </template>
 
 <script lang="ts">
-import type { GridClient } from "@threefold/grid_client";
 import { ref } from "vue";
 
 import { manual } from "@/utils/manual";
 
 import { useProfileManagerController } from "../components/profile_manager_controller.vue";
-import { useGrid } from "../stores";
+import { useProfileManager } from "../stores";
 import { createCustomToast, ToastType } from "../utils/custom_toast";
-import { updateGrid } from "../utils/grid";
+import { getGrid } from "../utils/grid";
 
 export default {
   name: "FundsCard",
   setup() {
     const loadingAddTFT = ref(false);
     const ProfileManagerController = useProfileManagerController();
-    const gridStore = useGrid();
-    const grid = gridStore.client as GridClient;
+    const profileManager = useProfileManager();
```

Comment:
Do we use profileManagerController now instead?

Reply:
Profile manager controller instead of what? The profile manager and profile manager controller have 2 different functionalities.